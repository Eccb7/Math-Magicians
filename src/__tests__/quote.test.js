import React from 'react';
import { render } from '@testing-library/react';
import QuoteDisplay from '../components/quote';

test('renders QuoteDisplay component correctly', () => {
  const { container } = render(<QuoteDisplay />);
  expect(container).toMatchSnapshot();
});
