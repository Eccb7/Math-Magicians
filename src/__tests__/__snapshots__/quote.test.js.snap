// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders QuoteDisplay component correctly 1`] = `
<div>
  <div
    class="centered-content"
  >
    <div>
      Loading...
    </div>
  </div>
</div>
`;
